Polygon_mesh_processing/test/Polygon_mesh_processing/remeshing_test_P_SM_OM.cpp

```diff
@@ -0,0 +1,80 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <CGAL/Surface_mesh.h>
+#include <CGAL/Polyhedron_3.h>
+#include <fstream>
+#include <map>
+#include <OpenMesh/Core/IO/MeshIO.hh>
+#include <OpenMesh/Core/Mesh/PolyMesh_ArrayKernelT.hh>
+#include <OpenMesh/Core/Mesh/TriMesh_ArrayKernelT.hh>
+#include <CGAL/boost/graph/graph_traits_PolyMesh_ArrayKernelT.h>
+#include <CGAL/boost/graph/graph_traits_TriMesh_ArrayKernelT.h>
+#include <CGAL/Polygon_mesh_processing/remesh.h>
+
+namespace PMP = CGAL::Polygon_mesh_processing;
+
+typedef CGAL::Exact_predicates_inexact_constructions_kernel Epic;
+
+
+int main()
+{
+  {
+    typedef CGAL::Surface_mesh<typename Epic::Point_3> SM;
+    SM sm;
+
+    std::ifstream in("data/elephant.off");
+    in >> sm;
+    PMP::isotropic_remeshing(faces(sm),
+                             0.02,
+                             sm);
+    std::ofstream out("sm.off");
+    out << sm << std::endl;
+  }
+
+  {
+  typedef CGAL::Polyhedron_3<Epic> P;
+  std::map<boost::graph_traits<P>::face_descriptor, std::size_t> fim;
+  P p; 
+  std::ifstream in("data/elephant.off");
+  in >> p;
+  PMP::isotropic_remeshing(faces(p),
+                             0.02,
+                           p,
+                           PMP::parameters::face_index_map(boost::make_assoc_property_map(fim)));
+    std::ofstream out("p.off");
+    out << p << std::endl;
+  }
```

Comment:
I was about to say that there is an indentation issue. But actually I think that this test should be removed, and the test `remeshing_test.cpp` should be modified so that it is run trice (of twice, depending on OpenMesh), for different types.